t/get_wildcard.t.proposed

```diff
@@ -0,0 +1,28 @@
+use t::Helper;
```

Comment:
Why is it called proposed? I would rather have a failing test, than a test that is not run.